src/util/formatting.c

```diff
@@ -78,9 +78,24 @@ struct tm* localtime_r(const time_t* t, struct tm* date) {
 	return date;
 #elif defined(PSP2)
 	return sceKernelLibcLocaltime_r(t, date);
+#elif defined(__CELLOS_LV2__)
+	date = localtime(t);
```

Comment:
```suggestion
	*date = *localtime(t);
```

Reply:
tested....OK..cause this:

```
ifeq  (,$(findstring win,$(platform)))
ifeq  (,$(findstring mingw,$(platform)))
ifneq ($(platform), vita)
ifneq ($(platform), ps3)
RETRODEFS += -DHAVE_LOCALTIME_R
endif
endif
endif
endif
```
in Makwfile.common

Reply:
wait for this:
	```
date = localtime(t);
	*date = *localtime(t);
```

Reply:
Well, so long as it builds, then that one suggestion I posted should be all that's needed.

Reply:
Now it's correct.